doc/sphinx/symlinks.py

```diff
@@ -5,13 +5,35 @@
 
 import os
 
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+
+def get_arguments(args=None):
+    """
+    Get arguments from command line
+    """
+    parser = ArgumentParser(prog='symlinks.py',
+                            usage='%(prog)s [options] ',
+                            description=''' %(prog)s  creates symlinks from doc/release/* to doc/sphinx/source/*''',
+                            formatter_class=ArgumentDefaultsHelpFormatter
+                            )
+    parser.add_argument('--clean', action='store_true', help='Remove all dynamically generated symlinks')
+
+    opts = parser.parse_args(args)
+    return opts
+
+
+
 
 def extfilter(directory, extension='.rst'):
     """Filter out files without extension '.rst'"""
     return [os.path.join(directory, f) for f in os.listdir(directory) if os.path.splitext(f)[1] == extension]
 
+def cleansymlinks():
```

Comment:
Nope, just a remnant of me brainstorming good ways to implement an `undo symlinks`. Removed in last push.